rosbag2_transport/include/rosbag2_transport/recorder.hpp

```diff
@@ -85,6 +85,11 @@ class Recorder : public rclcpp::Node
   ROSBAG2_TRANSPORT_PUBLIC
   void record();
 
+  /// @brief Stopping recording and closing writer.
```

Comment:
Should comment on `pause()` that it leaves the writer open.

Reply:
@james-rms I've updated Doxygen comments for `stop()` and `pause()` API calls. Please check it out.